index.html

```diff
@@ -26,7 +26,7 @@
     <a href="#" class="navbar__logo">Ahmed</a>
     <nav>
```

Comment:
- [ ] **Required:** Your site has passed so many tests, however, there are a few tests to be passed. Kindly use this [tool to check the accessibility of your website.](https://wave.webaim.org/). The tools also give suggestions on how to improve the page to pass all tests.

Reply:
Hi @brytebee,
Thanks for your review and I would like to ask about the contrast issues. I follow a specific design template in Figma and it has its own color decisions. Do I still have to change it and make my own decisions to improve accessibility when it comes to "contrast"?